### ISSUE TITLE
Cannot select a qualification for a member

### ISSUE THREAD

**Issue by plex3r** [fabXlabs/fabX#464]
When I try to add a qualification to a member record it doesn't display correctly

![Screenshot 2023-12-10 211831](https://github.com/fabXlabs/fabX/assets/4057454/95c29834-9a78-42c2-a295-f2dd7b983fb1)

**Comment by sschaeffner** [comment]
Hi,
probably our documentation has some need for improving the descriptions of the roles. I'll try my best here and update the documentation later.

The general ideas regarding qualifications are:
* A user with admin rights can create qualifications.
* A user with admin rights can make another member an instructor for a certain qualification
* A user with instructor rights for a qualification (and _only_ these users/members) can give out member qualifications

I assume you are logged in as either the automatically created admin or as another user with admin privileges. If you want to give the "CO2 Laser Cutters" qualification to the user whose page you're on, you have to give "yourself" (the user you're currently logged in as) the "CO2 Laster Cutters" instructor qualification. I.e.,

1. Go to the users list
2. Click on your the user you're logged in as
3. Find the **instructor** qualifications row and add the "CO2 Laser Cutters" qualification (by clicking the drop-down arrow right next to "Add"). It should show up as in the screenshot above.
4. Now, go to the users list, again
5. Click on the user you want to have this qualification as a member (i.e. to be able to use the tool it gives them access rights to)
6. Find the **member** qualification row and add the "CO2 Laser Cutters" qualification (by clicking the drop-down arrow right next to "Add"). It should show up as a tag right next to the "member qualifications" text.

To summarise: The admin privileges only allow you to make a user an instructor for a qualification. The user with the instructor rights then can actually hand out this qualification to members. Then, the member is able to use the tools associated with this qualification.

**Comment by plex3r** [comment]
Got it thanks I think I assumed an admin wouldn't need to be an instructor but I then worked through it last night late after I posted.

Thanks for confirming

**Comment by sschaeffner** [comment]
Follow-up: I've added some documentation on the different roles: https://github.com/fabXlabs/fabX/blob/main/docs/roles.md
